shared/profile/post-proof.desktop.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+import React from 'react'
+import {Box, Text, Button, Icon, PlatformIcon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './post-proof'
+
+const Render = (props: Props) => {
+  const {platform, platformUserName, platformSubtitle, descriptionView, descriptionText, proofText, proofAction, proofActionIcon, proofActionText, noteText, onCancel, onCancelText, onComplete, onCompleteText, isOnCompleteWaiting, errorMessage} = props
+  return (
```

Comment:
Although in this case I think you can get away with doing `onClick={onFoo}` instead of `onClick={() => onFoo()}` and avoid that flow trickiness altogether